build/package/windows/nri-386-installer/Product.wxs

```diff
@@ -1,96 +1,99 @@
-﻿<?xml version="1.0" encoding="UTF-8"?>
-<Wix xmlns="http://schemas.microsoft.com/wix/2006/wi">
-    <?define Version = "{IntegrationVersion}" ?>
-    <?define ProductId = "*" ?>
-    <?define UpgradeCode = "{27cf7682-b909-4940-8368-ce4719af1b82}" ?>

```

Comment:
I do not understand why git show them as if they were completely rewritten, is it due to end of line changes?

Reply:
mmm looks like is the name of the file. i will dobule check this . looks wrong . Thanks

Reply:
so github detects the file had move . if you see the file has changed (i cheked and looks .wxs is [fine](https://wixtoolset.org/documentation/manual/v3/votive/votive_item_templates.html) and also the msi is being generated correctly).